### PR TITLE
Merge/master 3

### PR DESCRIPTION
**Pull request by ashleyfae**
Merges in the `master` branch.

This is missing this fix:
https://github.com/easydigitaldownloads/easy-digital-downloads/blob/master/includes/payments/class-payment-stats.php#L190-L215
https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7507

Still investigating whether or not it's needed. If so, some refactoring will be needed.

### CODE REVIEW

**Comment by ashleyfae** [comment]
Note to self: replace `EDD_DB_Customers` with `EDD\Compat\Customer` and test.

**Comment by ashleyfae** [comment]
Regarding https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7507 -

The report screen looks correct:

Screenshot from 2020-05-15 12-55-56

Not sure about the admin column shown here:

issue-7807-edd-2-9

That screenshot was taken on `2.9.20` version tag. I was going to recount earnings to see if it was corrected in 3.0 but the tool looks to be broken (unrelated). Making an issue for that separately.

**Comment by ashleyfae** [comment]
I don't really have any specific testing steps/instructions... other than does anything look seriously broken/wrong? lol

File: includes/emails/functions.php [hunk 1]
```diff
@@ -376,3 +376,192 @@ function edd_sendwp_disconnect () {
...
+		activate_plugin( 'jilt-for-edd/jilt-for-edd.php' );
+	}
+
+	/*
+	* Final check to see if Jilt is available.
+	*/
+	if ( ! class_exists( 'EDD_Jilt_Loader' ) ) {
+		wp_send_json_error(
+			array(
+				'error' => __( 'Something went wrong. Jilt was not installed correctly.', 'easy-digital-downloads' ),
```
Comment (robincornett) [inline includes/emails/functions.php:448]: Noting that I'm getting this error message, although Jilt is actually installed. I think this is related to having 3.0 on a multisite install, though, not an issue with this PR.

**Review by robincornett — APPROVED** [review]

File: includes/emails/functions.php (hunk 1 above)
Comment (ashleyfae) [reply includes/emails/functions.php:448]: Can you try reproducing in `master` branch?

Comment (robincornett) [reply includes/emails/functions.php:448]: Ah, thanks, of course I should have done that. Interesting results: have done this several times in multisite with the `master` branch and it's about 50/50 as to whether it installs properly and redirects to Jilt or if the error message happens.

The SendWP install has gone properly in both 2.9 and 3.0.

**Review by spencerfinnell — APPROVED** [review]

**Pull request merged**